features/frameworks/utest/source/utest_harness.cpp

```diff
@@ -172,6 +172,7 @@ void Harness::raise_failure(const failure_reason_t reason)
     utest::v1::status_t fail_status = STATUS_ABORT;
     if (handlers->test_failure) handlers->test_failure(failure_t(reason, location));
     if (handlers->case_failure) fail_status = handlers->case_failure(case_current, failure_t(reason, location));
```

Comment:
Should we still be calling the failure handler if you want to ignore the failure ? 
From memory this fix looks too simple to do what you think it is going to.....

Reply:
What I wanted was the following:
1. Print the line calling this macro.
2. Exit the test in a success return code (which is what ignore should do).
Now, the current failure handler does just that, with the exception of failing the test in the ignore state. Hence the fix.